src/components/LinkListItem/LinkListItem.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { ListItemIcon, ListItemText } from '@material-ui/core';
+import styles from '../../App.module.sass';
+import EditableText from '../common/containers/LeftPanel/EditableTextListItem';
+
+export default function LinkListItem(props) {
+  const {
+    path, text, icon
+  } = props;
+
+  let iconComponent = '';
+
+  if (icon) {
+    iconComponent = (
+      <ListItemIcon>
+        {' '}
+        {` ${icon} `}
+        {' '}
+      </ListItemIcon>
+    );
+  }
+
+  let listItem;
+
+  if (text.editable) {
+    listItem = <EditableText text={text} />;
+  } else {
+    listItem = (
+      <>
```

Comment:
use `React.Fragment` if no div is setted

Reply:
eslint marks React.Fragment as an error :S 
do I have it misconfigured?
![image](https://user-images.githubusercontent.com/11907665/71551211-71ec6980-29c0-11ea-8872-d7ea9c061f7a.png)

Reply:
eslint configured properly!